app/views/requests/show.html.erb

```diff
@@ -1,39 +1,45 @@
-<p id="notice"><%= notice %></p>
+<!-- <p id="notice"><%= notice %></p> -->
```

Comment:
Why is this commented and not removed? :-)

Reply:
I was not sure if someone wants to use it and only commented it